src/types/license.types.ts

```diff
@@ -1,20 +1,50 @@
-export default interface License {
-  identifier: string;
-  code: string;
-  description: string;
-  description_en?: string;
-  description_no?: string;
-  issuer: string;
+import * as LicenseImages from '../resources/images/licenses';
+
+export interface License {
   name: string;
-  name_en?: string;
-  name_no?: string;
-  url: string;
-  url_en?: string;
-  url_no?: string;
-  url_image: string;
-  time_created: Date;
+  description: string;
+  image: any;
 }
 
+export const licenses: License[] = [
```

Comment:
Det finnes et api for dette: https://api.creativecommons.org/docs/readme_dev.html
Fun facts: 
De anbefaler å bruke development-versjonen som ble sist oppdatert i januar 2010(!), OG de returnerer bare XML :D 
Så jeg tenker faktisk at hardkoding er bedre atm. Har vært nevnt at vi skal støtte enda flere lisenser post-MVP, så mulig det burde bli levert via en backend-tjeneste en gang i fremtiden